_runners/vault.py

```diff
@@ -73,7 +73,10 @@ def _get_policies(minion_id, config):
 
   policies = []
   for pattern in policyPatterns:
-    policies.append(pattern.format(**mappings))
+    try:
+      policies.append(pattern.format(**mappings))
+    except KeyError:
+      log.error('Could not resolve policy pattern {0}'.format(pattern))
```

Comment:
I see your point. Changed to warning - reasoning being that this could easily be a quite frustrating thing to not find when there is some spelling error or similar. Sounds ok?